autofixture/generators.py

```diff
@@ -664,3 +665,15 @@ class UUIDGenerator(Generator):
 
     def generate(self):
         return uuid.uuid4()
+
+
+# Geo
+class PointFieldGenerator(Generator):
+
+    latitude = 19.99
+    longitude = 73.78
+
+    def generate(self):
+        dec_lat = random.random() / 100
+        dec_lon = random.random() / 100
+        return Point(self.longitude + dec_lat, self.latitude + dec_lon)
```

Comment:
Since we want to have test data generated, it might be useful to create the whole possibility of results. So I think latitude should be possible between -90° and 90°, and longitude between -180° and 180°. That would ofcourse result in a many points placed directly inside the Pacific Ocean, but I don't see this as a problem.

What do you think?